### DISCUSSION TITLE
Make maps configurable per list

### DISCUSSION THREAD

**Discussion by jmaas-onoffice** [onOffice-Web-Org/oo-wp-plugin#328] [_Discarded pitches]
> Time tracking ticket: T#3253495

# Map is shown either everywhere or nowhere
In an estate list, if there is an estate with a published address, a map will be shown above the list:
![grafik](https://user-images.githubusercontent.com/90763364/190354043-2fcad519-8ee2-4071-acb7-ea497ca68d13.png)

Similarly, the map is shown on an estate's detail page, if the address is published:
![grafik](https://user-images.githubusercontent.com/90763364/190354475-a3f2ab6a-c3d7-489f-ba44-922f2f845a38.png)

Since this is coupled to whether the address is published, it is not possible to only show the map in the estate detail page. This is something that a customer seemed to asked for (T#3244833).

# Allow configuring the map per list and for the detail view
We can add an option that determines whether to show or hide the list for each estate list and the detail view. This would allow users to e.g. only show the map on the detail view, or only for certain lists.

I propose this as a 1-week project.

**Comment by dai-eastgate** [comment]
@jmaas-onoffice Could you please explain more specifically about requirement "We can add an option that determines whether to show or hide the list for each estate list and the detail view. This would allow users to e.g. only show the map on the detail view, or only for certain lists"?

**Reply by jmaas-onoffice** [reply]
This is not selected for the current cycle.
